### DISCUSSION TITLE
Split by average ratio for windows

### DISCUSSION THREAD

**Discussion by anhthang** [asmvik/yabai#821] [Q&A] [answered]
Currently, I'm using yabai with setting like `split_ratio = 0.5`, it's good almost use cases. But I'm using a widescreen and it's not really good if I open 3 windows (1 window with 50% and 2 windows with 25%) or more. 

That's better if we have a way to calculate the average number (eg: 0.33 for 3 windows, 0.25 for 4 windows...) and put it there.

If it's available, pls let me know how to do that. Thank you!

**Comment by asmvik** [comment, marked as answer]
You can do this manually, on demand, with `yabai -m space --balance`, or by adding `yabai -m config auto_balance on` to your  config

**Reply by anhthang** [reply]
thank you. it works for my use case.
